src/libs/ReportUtils.ts

```diff
@@ -8211,6 +8217,44 @@ function createDraftTransactionAndNavigateToParticipantSelector(transactionID: s
         (policy) => policy && policy.type !== CONST.POLICY.TYPE.PERSONAL && policy.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE,
     );
 
+    if (actionName === CONST.IOU.ACTION.CATEGORIZE) {
+        const activePolicy = getPolicy(activePolicyID);
+        if (activePolicy && activePolicy?.type !== CONST.POLICY.TYPE.PERSONAL && activePolicy?.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE) {
+            const policyExpenseReportID = getPolicyExpenseChat(currentUserAccountID ?? -1, activePolicyID ?? '-1')?.reportID ?? '-1';
+            IOU.setMoneyRequestParticipants(transactionID, [
+                {
+                    selected: true,
+                    accountID: 0,
+                    isPolicyExpenseChat: true,
+                    reportID: policyExpenseReportID,
+                    policyID: activePolicyID ?? '-1',
+                    searchText: activePolicy?.name,
+                },
+            ]);
+            Navigation.navigate(ROUTES.MONEY_REQUEST_STEP_CATEGORY.getRoute(actionName, CONST.IOU.TYPE.SUBMIT, transactionID, policyExpenseReportID));
+            return;
+        }
+        if (filteredPolicies.length === 0 || filteredPolicies.length > 1) {
```

Comment:
@Ollyws We should prioritize checking the active policy first. we should only move to this condition if the `activePolicy` is a `personal` policy.

Reply:
@Ollyws We should only move to this condition if the `activePolicy` is a `personal` policy.

Reply:
@Ollyws Yes.

Reply:
@Ollyws More detail is [here](https://github.com/Expensify/App/issues/52253#issuecomment-2474765292).

Reply:
> If your default workspace is of a “group” type (collect or control), show the categories list for that workspace.

@Ollyws I believe this case is correct because your default policy is a group workspace after you create the first workspace. To set it to another default workspace, you can set it in OD.